astropy/table/tests/test_mixin.py

```diff
@@ -131,15 +132,55 @@ def test_io_quantity_write(tmpdir):
         assert 'cannot write table with mixin column(s)' in str(err.value)
 
 
+def test_io_time_write(tmpdir):
+    """
+    Test that table with Time mixin column can be written by io.fits but
+    not by io.votable and io.misc.hdf5. Validation of the output is done.
+    Test that io.fits writes a table containing Time mixin columns that can
+    be considerably round-tripped (metadata scale, format).
+    """
+    t = QTable()
+    t['a'] = time.Time([1,2], format='cxcsec', scale='tai', location=EarthLocation(u.Quantity(-2446353.80003635, unit='m'),
```

Comment:
The locations get converted to xyz and will not recover exactly; I think for these examples it is fine to use locations that are integer number of km (i.e., use the x,y,z version).